includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -1493,4 +1504,94 @@ public static function print_dashboard_glance_styles() {
 		<?php
 	}
 
+	/**
+	 * Filters post row actions.
+	 *
+	 * @param array    $actions Row action links.
+	 * @param \WP_Post $post Current WP post.
+	 * @return array Filtered action links.
+	 */
+	public static function filter_post_row_actions( $actions, $post ) {
+		// Replace 'Trash' text with 'Forget'.
+		if ( isset( $actions['trash'] ) ) {
+			$actions['trash'] = sprintf(
+				'<a href="%s" class="submitdelete" aria-label="%s">%s</a>',
+				get_delete_post_link( $post->ID ),
+				/* translators: %s: post title */
+				esc_attr( sprintf( __( 'Forget &#8220;%s&#8221;', 'amp' ), $post->post_title ) ),
+				esc_html__( 'Forget', 'amp' )
+			);
+		}
+
+		if ( isset( $actions['delete'] ) ) {
+			$actions['delete'] = sprintf(
+				'<a href="%s" class="submitdelete" aria-label="%s">%s</a>',
+				get_delete_post_link( $post->ID, '', true ),
+				/* translators: %s: post title */
+				esc_attr( sprintf( __( 'Forget &#8220;%s&#8221; permanently', 'amp' ), $post->post_title ) ),
+				esc_html__( 'Forget Permanently', 'amp' )
+			);
+		}
+
+		return $actions;
+	}
+
+	/**
+	 * Filters table views for the post type.
+	 *
+	 * @param array $views Array of table view links keyed by status slug.
+	 * @return array Filtered views.
+	 */
+	public static function filter_table_views( $views ) {
+		// Replace 'Trash' text with 'Forgotten'.
+		if ( isset( $views['trash'] ) ) {
+			$status = get_post_status_object( 'trash' );
+
+			$views['trash'] = str_replace( $status->label, esc_html__( 'Forgotten', 'amp' ), $views['trash'] );
+		}
+
+		return $views;
+	}
+
+
+	/**
+	 * Filters messages displayed after bulk updates.
+	 *
+	 * @param array $messages    Bulk message text.
+	 * @param array $bulk_counts Post numbers for the current message.
+	 * @return array Filtered messages.
+	 */
+	public static function filter_bulk_post_updated_messages( $messages, $bulk_counts ) {
+		if ( get_current_screen()->id === sprintf( 'edit-%s', self::POST_TYPE_SLUG ) ) {
+			$messages['post'] = array_merge(
+				$messages['post'],
+				array(
+					/* translators: %s is the number of posts permanently forgotten */
+					'deleted'   => _n(
+						'%s invalid AMP page permanently forgotten.',
+						'%s invalid AMP post permanently forgotten.',
+						$bulk_counts['deleted'],
```

Comment:
Minor thing (and other such instances): this could be wrapped in `number_format_i18n()`.

Reply:
Meh, core isn't doing it. So no need for it here.